Src/Screens/ProfileScreen/Screens/Referrals/Viewables/ReferralsInvitationMemberIcon.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ReferralsInvitationMemberIcon.swift
+//  UITests
+//
+//  Created by Sam Pettersson on 2019-06-03.
+//
+
+import Foundation
+import Flow
+import UIKit
+import Form
+
+struct ReferralsInvitationMemberIcon {}
+
+extension ReferralsInvitationMemberIcon: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Disposable) {
+        let view = UIStackView()
+        view.axis = .horizontal
+        view.alignment = .center
+        
+        let styling = UIView()
```

Comment:
wouldn't `container` be a better name than `styling`?

Reply:
Renamed it to `boxView` instead :)